.github/workflows/tests.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
-      - name: Set up Java 8
+      - name: Set up Java 17
         uses: actions/setup-java@v2
         with:
-          java-version: '8'
+          java-version: '17'
```

Comment:
Is Java 17 required to compile card form now? Is this a breaking change then?

Reply:
It might be I don't see an explicit Java version specified in the `build.gradle`. I updated the build.gradle file to make it explicit.

Reply:
I guess I'm trying to understand - can we support Android 15 without bumping the java version? Or are we doing a major version of card form here? I more so am concerned that this change will force a major version in drop-in which I don't think we want to do yet

Reply:
Oh got it yeah I forgot DropIn doesn't use v5 yet. AGP uses JDK 17 to build, but we can set `sourceCompatability` to Java 8 and it should be fine.